resources/assets/js/label-trees/components/labelTrees.vue

```diff
@@ -242,6 +271,27 @@ export default {
         on(key, fn) {
             this.events.on(key, fn);
         },
+        moveLabelTrees(treeIdx, targetIdx) {
+            let customOrder = this.swapElements(this.treeIds, treeIdx, targetIdx);
+            this.customOrder = customOrder;
+
+            this.customOrderStorageKeys.forEach(function (storageKey) {
+                localStorage.setItem(storageKey, JSON.stringify(customOrder));
+            });
+        },
+        swapElements(arr, idx1, idx2) {
+            let element = arr.splice(idx1, 1)[0];
+            arr.splice(idx2, 0, element);
+            return arr;
+        },
+        changeCustomOrder() {
+            let customOrder = JSON.parse(
+                localStorage.getItem(this.customOrderStorageKey)
+            );
+            if (customOrder) {
+                this.customOrder = customOrder;
+            }
+        }
```

Comment:
This also seems to be used nowhere. Please check your changes for any other unused stuff.